src/Http/Headers/src/PublicAPI.Unshipped.txt

```diff
@@ -2,5 +2,6 @@
 *REMOVED*Microsoft.Net.Http.Headers.RangeConditionHeaderValue.RangeConditionHeaderValue(Microsoft.Net.Http.Headers.EntityTagHeaderValue? entityTag) -> void
 Microsoft.Net.Http.Headers.MediaTypeHeaderValue.MatchesMediaType(Microsoft.Extensions.Primitives.StringSegment otherMediaType) -> bool
 Microsoft.Net.Http.Headers.RangeConditionHeaderValue.RangeConditionHeaderValue(Microsoft.Net.Http.Headers.EntityTagHeaderValue! entityTag) -> void
+static Microsoft.Net.Http.Headers.CookieHeaderValue.TryParseIntoDictionary(Microsoft.Extensions.Primitives.StringValues inputs, System.Collections.Generic.Dictionary<string!, string!>! store, bool enableCookieNameEncoding) -> bool
```

Comment:
I'd rather not make this public, should I just do internals visible to?

Reply:
That's banned 😄

Reply:
IVT everything

Reply:
What do you mean?

Reply:
1 new public API?

Reply:
Referring to @pranavkm, idk what IVT means here.

I'd prefer to not have a public API for this, but don't know another way around it if we want use the static parser inside of CookieHeaderValue